### PR TITLE
faq component

### PR DESCRIPTION
**Pull request by alex13bright**

### CODE REVIEW

File: app/cms/zod/roomSchema.ts [hunk 1]
```diff
@@ -89,10 +91,14 @@ const pagesSchema = z
...
-            document_meta_title: z.string().optional(),
-            content: z.string().optional().default(''),
+            document_meta_description: z.string().nullable().default(''),
+            h1: z.string().nullable().default(''),
+            document_meta_title: z.string().nullable(),
+            content: z
+              .string()
+              .nullable()
+              .default('')
+              .transform((val) => val ?? ''),
```
Comment (alex-adamant) [inline app/cms/zod/roomSchema.ts:101]: Probably you don't need `default` here, this transform will catch both `null` and `undefined`.

Comment (alex-adamant) [reply app/cms/zod/roomSchema.ts:101]: Oh, but for other items `default` won't work either, right?

File: app/components/common/SocialMedia.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react'
+import { SocialMediaListType, SocialMediaDescType } from '~/core/types'
+import styled, { StyledComponent } from 'styled-components'
+import { pseudoIconCss, PseudoIconCss } from '~/styles/sharedCss'
+
+export const descs: SocialMediaDescType = {
+  telegram: { color: '#4b9fdb', url: (address) => `https://telegram.me/${address}` },
```
Comment (alex-adamant) [inline app/components/common/SocialMedia.tsx:7]: I'd prefer not to name methods with nouns. Maybe `getLink` or something like this?

File: app/components/common/SocialMedia.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react'
+import { SocialMediaListType, SocialMediaDescType } from '~/core/types'
+import styled, { StyledComponent } from 'styled-components'
+import { pseudoIconCss, PseudoIconCss } from '~/styles/sharedCss'
+
+export const descs: SocialMediaDescType = {
```
Comment (alex-adamant) [inline app/components/common/SocialMedia.tsx:6]: `descs` is not the best name in the world. Why don't just go with the full name? (same for style).

File: app/dynamic-content/components/Injection.tsx [hunk 4]
```diff
@@ -1,8 +1,48 @@
...
+      if (!alt || !src) continue
+      images.push({
+        title: alt,
+        url: src,
+        width: stripPx(width),
+        height: stripPx(height),
+      })
+    }
+  }
+  console.log(images)
```
Comment (alex-adamant) [inline app/dynamic-content/components/Injection.tsx:43]: A stray `console.log`

File: app/dynamic-content/components/custom/ContentLightbox.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+const Main = styled.div`
+  @media screen and ${widthAtLeast.md} {
+    border: 1px solid #e9e9e9;
+    background: #fafafa;
+    padding: 20px;
+    border-radius: 4px;
+  }
+`
+
+const galleryOptions = { zoom: false } as any
```
Comment (alex-adamant) [inline app/dynamic-content/components/custom/ContentLightbox.tsx:35]: Why don't types work here?

Comment (alex-adamant) [reply app/dynamic-content/components/custom/ContentLightbox.tsx:35]: Isn't this library typed?

**Review by alex-adamant — APPROVED** [review]
Changes looks good! I've left some comments/questions, but they are not blockers for me.

File: app/cms/zod/roomSchema.ts (hunk 1 above)
Comment (alex13bright) [reply app/cms/zod/roomSchema.ts:101]: yes, you right, default is redundant. I'll remove it

File: app/components/common/SocialMedia.tsx (hunk 2 above)
Comment (alex13bright) [reply app/components/common/SocialMedia.tsx:7]: I try to name that way too. It was not necessary exception.

File: app/dynamic-content/components/Injection.tsx (hunk 4 above)
Comment (alex13bright) [reply app/dynamic-content/components/Injection.tsx:43]: Yeah :)

File: app/dynamic-content/components/custom/ContentLightbox.tsx (hunk 5 above)
Comment (alex13bright) [reply app/dynamic-content/components/custom/ContentLightbox.tsx:35]: I should use type PhotoSwipeOptions but better question do why I even try to specify type there.

Comment (alex13bright) [reply app/dynamic-content/components/custom/ContentLightbox.tsx:35]: I can specify for editor hints...

**Pull request merged**